### PR TITLE
ID review #42

### PR DESCRIPTION
**Pull request by MaiHameed**

### CODE REVIEW

File: README.adoc [hunk 1]
```diff
@@ -190,16 +191,15 @@ Create a class to externalize your container configurations.
 `system/src/test/java/it/io/openliberty/guides/system/AppContainerConfig.java`
 ----
 
+The [hotspot=AppContainerConfig file=1]`AppContainerConfig` class externalizes test container setup and configuration, so
```
Comment (Charlotte-Holt) [inline README.adoc:194]: Should "test container" also be changed here to "Testcontainer"?

File: README.adoc [hunk 2]
```diff
@@ -45,18 +45,19 @@ image::reactive-messaging-system-inventory.png[Reactive system inventory applica
...
+the application’s lifecycle – from development, through testing, and into production.
 You can learn more about MicroShed Testing in the
 https://openliberty.io/guides/microshed-testing.html[Testing a MicroProfile or Jakarta EE application^] guide.
 
 == Additional prerequisites
 
-Install Docker (https://docs.docker.com/engine/installation[see the official Docker documentation^]) and start your Docker environment.
-You will build and run the microservices in Docker containers. An installation of Apache Kafka is provided in another Docker container.
+You need to have Docker installed. For installation instructions, refer to the
+(https://docs.docker.com/engine/installation[official Docker documentation^]) You will build and run the microservices in
```
Comment (Charlotte-Holt) [inline README.adoc:59]: I'd suggest updating 
`For installation instructions, refer to the (https://docs.docker.com/engine/installation[official Docker documentation^])`
To
`For installation instructions, refer to the official link:https://docs.docker.com/get-docker/[Docker documentation].`

File: README.adoc [hunk 3]
```diff
@@ -153,32 +153,33 @@ include::finish/inventory/pom.xml[]
...
 include::{common-includes}/devmode-start.adoc[]
 
 Now you can add your test files.
 
-The running `system` service searches for a Kafka topic to push its messages to. However, as there are not yet any running
-Kafka services, the `system` service throws some errors. Later in the guide, you will write and run tests that start a Kafka
-test container for the `system` service to communicate. This will resolve the errors that you see now.
+The running `system` service searches for a Kafka topic to push its messages to. Because there are not yet any running Kafka
+services, the `system` service throws errors. Later in the guide, you will write and run tests that start a Kafka
+Testcontainer that can communicate with the`system` service. This will resolve the errors that you see now.
```
Comment (Charlotte-Holt) [inline README.adoc:182]: Need a space between `the` and `system service`

File: README.adoc (hunk 1 above)
Comment (MaiHameed) [reply README.adoc:194]: In this case, it's referring to the Kafka Testcontainer as well as the application test container so I generalized both under test container.

**Pull request merged**